app/policies/ownership_policy.rb

```diff
@@ -1,7 +1,11 @@
 class OwnershipPolicy < ApplicationPolicy
   class Scope < Scope
     def resolve
-      scope.all
+      if rubygems_org_admin?
+        scope.all
+      else
+        scope.where(authorizer: user).or(scope.where(user: user))
+      end
```

Comment:
I don't really like that an admin using the site normally could see ownership's they wouldn't otherwise be able to. I think it makes sense for us to make sure admins only have access to admin views in the admin console.

Reply:
I do agree. I like the pattern to keep pages unified for all users and expose additional data only at special admin panel.I Thought admin user and common user are separated already. :thinking:

Reply:
I think they are, but as I'm reusing the pundit objects for normal user authZ, I'm noticing a blurring of lines that is easy to make. I want to figure out how to ensure this doesn't happen. Maybe there's a way to ensure that it can only approve admin access from avo?

Reply:
Maybe separate "user" and "admin user" policy scopes and use "admin user" ones in avo exclusively?

Reply:
If so, we could probably move all current policies to admin scope and then create new policies for normal users (this is probably the correct thing to do)

Reply:
I did this in #4745